openupgrade_scripts/scripts/website_sale/15.0.1.1/post-migration.py

```diff
@@ -1,6 +1,40 @@
+import re
+
 from openupgradelib import openupgrade
 
 
+def replace_content_in_product_custom_text(env):
```

Comment:
Put a more descriptive method name (and you can also put docstring):

```suggestion
def _extract_custom_product_page_term_conditions(env):
```